config/settings.yml

```diff
@@ -147,6 +147,7 @@ feature:
   price_policy_requires_note: true
   multi_facility_accounts: true
   facility_directors_can_manage_price_groups: true
+  multiple_statements_download: true
```

Comment:
Do we need this to be behind a FF? I don't remember any schools saying it could be a problem for them